src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -152,6 +209,38 @@ class ZHAConfigDashboard extends LitElement {
     this._configuration = await fetchZHAConfiguration(this.hass!);
   }
 
+  private async _fetchSettings(): Promise<void> {
+    this._networkSettings = await fetchZHANetworkSettings(this.hass!);
+  }
+
+  private async _createAndDownloadBackup(): Promise<void> {
+    this._generatingBackup = true;
+    const backup_and_metadata: ZHANetworkBackupAndMetadata =
+      await createZHANetworkBackup(this.hass!);
+    this._generatingBackup = false;
+
+    if (!backup_and_metadata.is_complete) {
+      await showAlertDialog(this, {
+        title: "Backup is incomplete",
+        text: "A backup has been created but it is incomplete and cannot be restored. This is a coordinator firmware limitation.",
+      });
```

Comment:
return to stop further executing this function.

```suggestion
      });
      return;
```

Reply:
It's intentional. An incomplete backup still has useful information, just not the network key.